andela/mmdp_site
# MMDP 694 Filter stakeholders by location bug fix

**What does this PR do?**

fixes stakeholder endpoint returning beneficiaries located in LGAs not within specified state

**description of Task to be completed?**

- filter beneficiaries by state

**How should this be manually tested?**

- `git fetch`
- `git checkout bg-fix-filter-stakeholder-by-location-returning-data-from-other-locations-MMDP-694`
- `yarn dev`
-  use postman to hit `http://0.0.0.0:3000/api/v1/location?state=Edo&focusAreaName` 
- notice that for each stakeholder, under the beneficiaries dictionary, communities returned are now only from the said state

**Any background context you want to provide?**

This bug fix affects 3 tables on our site, whenever one wishes to see information pertinent to a certain state, the data displayed shows data from other states

**What are the relevant Jira issues?**

[MMDP-694](https://viisaus.atlassian.net/browse/MMDP-694)

**Screenshots (if appropriate)**

N/A

**Questions:**

N/A
